service/src/test/java/com/sedliarov/learningtable/utils/StudentFixture.java

```diff
@@ -0,0 +1,77 @@
+package com.sedliarov.learningtable.utils;
```

Comment:
Check my exaple in more details https://github.com/KrlSed/lern-table/pull/3/files#r964088307

Reply:
Done